pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -34,8 +36,9 @@ stdenv.mkDerivation rec {
     nuget sources Add -Name tmpsrc -Source $TMP/nuget
     nuget init ${linkFarmFromDrvs "deps" deps} $TMP/nuget
 
-    dotnet restore --source $TMP/nuget BTCPayServer/BTCPayServer.csproj
-    dotnet publish --no-restore --output $out/share/$pname -c Release BTCPayServer/BTCPayServer.csproj
+    # RazorCompileOnBuild needed for altcoin support
+    dotnet restore --source $TMP/nuget ${optionalString altcoinSupport ''/p:Configuration="Altcoins-Release" /p:RazorCompileOnBuild=false''} BTCPayServer/BTCPayServer.csproj
+    dotnet publish --no-restore --output $out/share/$pname ${optionalString altcoinSupport "-c Altcoins-Release"} BTCPayServer/BTCPayServer.csproj
```

Comment:
```suggestion
    dotnet restore --source $TMP/nuget ${lib.optionalString altcoinSupport ''/p:Configuration="Altcoins-Release" /p:RazorCompileOnBuild=false''} BTCPayServer/BTCPayServer.csproj
    dotnet publish --no-restore --output $out/share/$pname ${lib.optionalString altcoinSupport "-c Altcoins-Release"} BTCPayServer/BTCPayServer.csproj
```

Reply:
Fixed in bf6cac3e7a94de1a5dc0da81899ec489780210a7